### DISCUSSION TITLE
Programmatic Naming of Colors from Params

### DISCUSSION THREAD

**Discussion by cgrobb** [meodai/color-names#135] [General]
kettlecorn on Twitter, maker of [bloom3d.com](https://t.co/eot0pEir2C), is doing programmatic naming of colors from params in support of accessibility:

https://twitter.com/kettlecorn/status/1500341317569855488?s=20&t=BmMmFf0mDvntO7-bUG-qvQ

Code (Rust):
https://gist.github.com/kettle11/666edd2645088392582950a8e471c7be

**Comment by meodai** [comment]
Hello @cgrobb 
Thanks for sharing this! 
All the possible outputs from this, are probably already in the list. 

I have a project of my own that aims to do the same that kettlecorn did: https://words.github.io/color-description/
What I like about his implementation, is that he is using okLAB!
